horovod/tensorflow/gradient_aggregation.py

```diff
@@ -1,4 +1,7 @@
 import tensorflow as tf
+from distutils.version import LooseVersion
+
+_IS_TF2 = LooseVersion(tf.__version__) >= LooseVersion('2.0.0')
```

Comment:
Here's one left-over usage of `distutils` that should also be replaced by `packaging.version.parse`. 

All of distutils will be removed in Python 3.12.